Sources/Purchasing/Offering.swift

```diff
@@ -92,11 +92,18 @@ import Foundation
 
     public override var description: String {
         return """
-        <Offering {\n\tidentifier=\(identifier)\n\tserverDescription=\(serverDescription)\n"
-        \tavailablePackages=\(valueOrEmpty(availablePackages))\n\tlifetime=\(valueOrEmpty(lifetime))\n
-        \tannual=\(valueOrEmpty(annual))\n\tsixMonth=\(valueOrEmpty(sixMonth))\n
-        \tthreeMonth=\(valueOrEmpty(threeMonth))\n\ttwoMonth=\(valueOrEmpty(twoMonth))\n
-        \tmonthly=\(valueOrEmpty(monthly))\n\tweekly=\(valueOrEmpty(weekly))\n}>
+        <Offering {
+            identifier=\(self.identifier)\n\tserverDescription=\(self.serverDescription)"
```

Comment:
Could we also separate the `serverDescription` to a new line?

Reply:
Oh thanks I missed that